homeassistant/components/mqtt/select.py

```diff
@@ -114,27 +121,29 @@ class MqttSelect(MqttEntity, SelectEntity, RestoreEntity):
     """representation of an MQTT select."""
 
     _entity_id_format = select.ENTITY_ID_FORMAT
-
     _attributes_extra_blocked = MQTT_SELECT_ATTRIBUTES_BLOCKED
-
-    def __init__(self, hass, config, config_entry, discovery_data):
+    _templates: dict[str, Callable[..., Any]]
```

Comment:
Maybe this should be split in two?
```suggestion
    _command_templates: dict[str, Callable[[PublishPayloadType], PublishPayloadType]]
    _value_templates: dict[str, Callable[[ReceivePayloadType], ReceivePayloadType]]
```